Hypeclass/Source/DancerDetailViewController.swift

```diff
@@ -0,0 +1,335 @@
+            dancerInfoStackView.topAnchor.constraint(equalTo: profileImageView.topAnchor, constant: 10),
+            
+            instagramImageView.leadingAnchor.constraint(equalTo: profileImageView.trailingAnchor, constant: 20),
+            instagramImageView.bottomAnchor.constraint(equalTo: profileImageView.bottomAnchor, constant: -10),
+            instagramImageView.heightAnchor.constraint(equalToConstant: 22),
+            instagramImageView.widthAnchor.constraint(equalToConstant: 22),
+
+            youtubeImageView.leadingAnchor.constraint(equalTo: instagramImageView.trailingAnchor, constant: 10),
+            youtubeImageView.bottomAnchor.constraint(equalTo: profileImageView.bottomAnchor, constant: -10),
+            youtubeImageView.heightAnchor.constraint(equalToConstant: 22),
+            
+            scheduleContentLabel.leadingAnchor.constraint(equalTo: dancerDetailContentView.leadingAnchor, constant: 25),
+            scheduleContentLabel.topAnchor.constraint(equalTo: profileImageView.bottomAnchor, constant: 25),
+            
+            scheduleCollectionView.leadingAnchor.constraint(equalTo: dancerDetailContentView.leadingAnchor,constant: 20),
+            scheduleCollectionView.trailingAnchor.constraint(equalTo: dancerDetailContentView.trailingAnchor, constant: -20),
+            scheduleCollectionView.topAnchor.constraint(equalTo: scheduleContentLabel.bottomAnchor, constant: 10),
+            scheduleCollectionView.heightAnchor.constraint(equalToConstant: 160),
+            
+            separatorLine.leadingAnchor.constraint(equalTo: scheduleCollectionView.leadingAnchor,constant: 0),
+            separatorLine.trailingAnchor.constraint(equalTo: scheduleCollectionView.trailingAnchor,constant: 0),
+            separatorLine.topAnchor.constraint(equalTo: scheduleCollectionView.topAnchor,constant: 60),
+            separatorLine.bottomAnchor.constraint(equalTo: scheduleCollectionView.topAnchor,constant: 61),
+            
+            videoContentLabel.leadingAnchor.constraint(equalTo: dancerDetailContentView.leadingAnchor, constant: 25),
+            videoContentLabel.topAnchor.constraint(equalTo: scheduleCollectionView.bottomAnchor, constant: 25),
+            
+            mockUpThumNail.leadingAnchor.constraint(equalTo: dancerDetailContentView.leadingAnchor, constant: 25),
+            mockUpThumNail.topAnchor.constraint(equalTo: videoContentLabel.bottomAnchor, constant: 10),
+            
+            introduceContentLabel.leadingAnchor.constraint(equalTo: dancerDetailContentView.leadingAnchor, constant: 25),
+            introduceContentLabel.topAnchor.constraint(equalTo: mockUpThumNail.bottomAnchor, constant: 25),
+            
+            introduceContent.leadingAnchor.constraint(equalTo: dancerDetailContentView.leadingAnchor, constant: 25),
+            introduceContent.topAnchor.constraint(equalTo: introduceContentLabel.bottomAnchor, constant: 25)
```

Comment:
더불어서 지금 모든 View 에 왼쪽 오른쪽 25 마진이 들어가는 데, 
이렇게되면 양옆 마진을 수정하고자할 때, 일일히 수정해주어야합니다.
가능한 이러한 양옆 마진을 동일하게 적용받는 View 끼리 제약조건을 걸어주세요 
아래는 제안입니다!
```suggestion
            videoContentLabel.leadingAnchor.constraint(equalTo: dancerDetailContentView.leadingAnchor, constant: 25),
            videoContentLabel.topAnchor.constraint(equalTo: scheduleCollectionView.bottomAnchor, constant: 25),
            
            mockUpThumNail.leadingAnchor.constraint(equalTo: videoContentLabel.leadingAnchor),
            mockUpThumNail.topAnchor.constraint(equalTo: videoContentLabel.bottomAnchor, constant: 10),
            
            introduceContentLabel.leadingAnchor.constraint(equalTo: mockUpThumNail.leadingAnchor),
            introduceContentLabel.topAnchor.constraint(equalTo: mockUpThumNail.bottomAnchor, constant: 25),
            
            introduceContent.leadingAnchor.constraint(equalTo: introduceContentLabel.leadingAnchor),
            introduceContent.topAnchor.constraint(equalTo: introduceContentLabel.bottomAnchor, constant: 25)
```

Reply:
예시를 보니 확실하게 이해가 됐어요! 감사합니다! 더 좋은 유지보수 방법이네요!